src/components/Theme/constants.css

```diff
@@ -4,6 +4,35 @@
 @large-text-margin: 40px;
 @small-text-margin: 24px;
 
+/* Solid colors */
+@color-accent-primary: #00C8A0;
+@color-accent-secondary: #00C9A1;
+@color-text-dark: #2C3E4F;
+@color-camera-error-overlay: #1C1F21;
+@color-white: #ffffff;
+@color-black: #000000;
+@color-small-print: #0F2536;
+@color-background: #D8D8D8;
+@color-background-secondary: #f3f3f4;
+@color-button-text: #585E5F;
+@color-button-border: #8A9293;
+@color-tips-pill: #5D6B78;
+@color-input-border: #667080;
+@color-selectors-border: #8B9094;
+@color-hint: #565B5E;
+@color-spinner: #ccc;
+@color-error: #DC2A2A;
+@color-warning: #2A7AB0;
+
+/* Transparent colors */
+@color-transparent: transparent;
```

Comment:
I will keep the variable for now because I don't know yet if we will have to replace the colour of those elements yet. If that's the case I will find a better variable name later on.